Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -682,81 +664,90 @@ namespace gdjs {
           });
         }
 
+        const lastScoreSavingState = getLastScoreSavingState({
+          hasSucceeded: true,
+        });
+        const lastSavedLeaderboardEntryId = lastScoreSavingState
+          ? lastScoreSavingState.lastSavedLeaderboardEntryId
+          : null;
+
         const gameId = gdjs.projectData.properties.projectUuid;
         const isDev = runtimeScene
           .getGame()
           .isUsingGDevelopDevelopmentEnvironment();
         const targetUrl = `https://gd.games/games/${gameId}/leaderboard/${leaderboardId}?inGameEmbedded=true${
           isDev ? '&dev=true' : ''
+        }${
+          lastSavedLeaderboardEntryId
+            ? `&playerLeaderboardEntryId=${lastSavedLeaderboardEntryId}`
+            : ''
```

Comment:
I wonder if we should use URLSearchParams, it's starting to look a bit messy here

Reply:
Good idea, will do.